scripts/release.sh

```
@@ -149,6 +453,76 @@ EOF
+                    arguments=$(echo "$arguments" | jq '. + {since_tag: null}')
+                fi
+            fi
+            
+            local result
+            result=$(execute_function_call "$function_name" "$arguments")
+            
+            # Properly escape the result for JSON
+            local escaped_result
+            escaped_result=$(printf '%s' "$result" | jq -R -s .)
+            
+            function_results+=('{"tool_call_id": "'$call_id'", "role": "tool", "content": '$escaped_result'}')
```

Shellcheck (0.10.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)